std/uni.d

```diff
@@ -9108,7 +9108,15 @@ bool isMark(dchar c)
 @safe pure nothrow @nogc
 bool isNumber(dchar c)
 {
-    return numberTrie[c];
+    // optimization
+    if (c < 0xAA)
```

Comment:
I know. This was a suggestion to improve the code style, if there is no negative perf difference. IMO, it may be even something like 0.001% faster (one less comparison for the ASCII digit case), but I haven't tested.